homeassistant/config_entries.py

```diff
@@ -1462,6 +1462,11 @@ def async_get_entry(self, entry_id: str) -> ConfigEntry | None:
         """Return entry with matching entry_id."""
         return self._entries.data.get(entry_id)
 
+    @callback
+    def async_entry_ids(self) -> list[str]:
+        """Return entry ids."""
+        return list(self._entries.data)
```

Comment:
any reason to turn this into a list and not use `.keys()` like entity registry ?

Reply:
Since integrations will likely iterate it I was worried it would change during iteration.

I wrapped the `async_` ones in `list` to avoid that and left the views for the underlying objects in registries